package.json

```diff
@@ -54,8 +54,8 @@
     "@babel/preset-env": "^7.22.20",
     "@babel/preset-react": "^7.22.15",
     "@babel/preset-typescript": "^7.23.0",
-    "@microsoft/api-documenter": "^7.23.9",
-    "@microsoft/api-extractor": "^7.38.0",
+    "@microsoft/api-documenter": "~7.23.38",
```

Comment:
Can we add a comment about why this one is pinned?

Reply:
package.json does not support comments. There are some hacky (and ugly) workarounds that only work in some specific cases. AFAIK, there is no good way to add comments to "dependencies" and "devDependencies". Technically, this trick works:
```json
  "comments": {
    "devDependencies": {
      "@microsoft/api-documenter": "comment here"
    }
  },
```
I really don't like it. I'll only use it as a last resort.

Reply:
Bleh, you're right, that's JSON for you.

I guess I'll be satisfied with blame pointing to this discussion.